src/it/scala/djinni/GeneratorIntegrationTest.scala

```diff
@@ -412,6 +412,33 @@ class GeneratorIntegrationTest extends IntegrationTest with GivenWhenThen {
       assertFileExists(s"$outputPath/AllDatatypes.mm")
     }
 
+    it(
+      "should be able to include objc generic type information for extern types"
+    ) {
+      Given(
+        "an IDL-file that uses a parameterized extern type that enables Objective-C option 'generics: true'"
+      )
+      val idlFile = "my_record_objc_generics"
+      When("generatinc the Objective-C gluecode")
```

Comment:
Sry for being picky 🙈 

```suggestion
        "an IDL-file that uses a parameterized extern type that enables the Objective-C option 'generics: true'"
      )
      val idlFile = "my_record_objc_generics"
      When("generating the Objective-C gluecode")
```